queue-health/graph/graph.py

```diff
@@ -33,6 +33,8 @@
 import matplotlib.pyplot as plt
 import numpy
 
+from pprint import pprint
```

Comment:
Import modules not functions

https://google.github.io/styleguide/pyguide.html#Imports